internal/service/connect/hours_of_operation.go

```diff
@@ -0,0 +1,342 @@
+package connect
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/connect"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+)
+
+func ResourceHoursOfOperation() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceHoursOfOperationCreate,
+		ReadContext:   resourceHoursOfOperationRead,
+		UpdateContext: resourceHoursOfOperationUpdate,
+		DeleteContext: resourceHoursOfOperationDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(connectHoursOfOperationCreatedTimeout),
+			Delete: schema.DefaultTimeout(connectHoursOfOperationDeletedTimeout),
+		},
+		Schema: map[string]*schema.Schema{
+			"config": {
+				Type:     schema.TypeList,
+				Required: true,
+				MinItems: 0,
+				MaxItems: 100,
```

Comment:
why include the `MaxItems` & `Min` hard limit?

Reply:
Limits for `Config` discovered by @AdamTylerLynch in an earlier review

Reply:
Are these service quotas?

Reply:
@drewmullen from https://console.aws.amazon.com/servicequotas/home, the `Hours of operation per instance` is listed at 100 and is adjustable

Reply:
👍 the terraform provider should not impose hard limits on service quotas - thats what the svc quotas are for :) so this can be removed

Reply:
Hi @drewmullen, I believe the limit for the service itself (hours of operation) is 100 (soft limit) but there's also a hard limit on the `conifg` attribute of the service which these limits are for and @AdamTylerLynch pointed out in an earlier review. I checked manually by adding 100 `Configs` in the console. At the 101th entry I get get an error on the `config` attribute.

Reply:
can we just get a comment in the code here explaining why theres a hard limit? specifically what is the scope of this effect - does an account only get 100? is there a way to reset the 100?